Measurements/Tests.qs

```diff
@@ -22,9 +22,10 @@ namespace Quantum.Kata.Measurements {
 
     // "Framework" operation for testing single-qubit tasks for distinguishing states of one qubit
     // with Bool return
-    operation DistinguishTwoStates_OneQubit (statePrep : ((Qubit, Int) => Unit), testImpl : (Qubit => Bool)) : Unit {
+    operation DistinguishTwoStates_OneQubit (statePrep : ((Qubit, Int) => Unit), testImpl : (Qubit => Bool), str1 : String, str_aim : String) : Unit {
```

Comment:
I'd pass string representations as an array, indexed by the same integer as used to pass to `statePrep` - this way it will be harder to confuse which string corresponds to which representation.

Reply:
Working on this one, thanks for the suggestion
Variable will be:     stateName : String[]